cadquery/occ_impl/exporters/dxf.py

```diff
@@ -76,16 +76,23 @@ def _dxf_spline(e: Edge, msp: ezdxf.layouts.Modelspace, plane: Plane):
         curve, adaptor.FirstParameter(), adaptor.LastParameter(), CURVE_TOLERANCE
     )
 
+    # need to apply the transform on the geometry level
+    spline.Transform(plane.fG.wrapped.Trsf())
+
     order = spline.Degree() + 1
     knots = list(spline.KnotSequence())
     poles = [(p.X(), p.Y(), p.Z()) for p in spline.Poles()]
+    weights = (
+        [spline.Weight(i) for i in range(1, spline.NbPoles() + 1)]
+        if spline.IsRational()
+        else None
+    )
 
-    weights = list(spline.Weights()) if spline.IsRational() else None
+    if spline.IsPeriodic():
```

Comment:
I'm not sure if this is needed, but it was hand-tested on Periodic splines. Somehow GeomConvert.SplitBSplineCurve_s seems to remove the periodicity, but I'm not sure if it'll be always the case